pages/charcuterie.js

```diff
@@ -75,6 +76,10 @@ export default function Charcuterie() {
           <h2>Footer</h2>
         </ContentContainer>
         <Footer />
+        <h2>Join Us Page</h2>
+        <div style={{ background: '#000', padding: '81px' }}>
+          <JoinUs hiring={true} deadlineDay={30} deadlineMonth="September"/>
+        </div>
```

Comment:
I would just stick the `JoinUs` page here without providing the custom styling.
Also with the recent addition of the Typography elements, you can refactor `h2` to use `TextTitle2`